src/encoded/static/components/experiment.js

```diff
@@ -291,6 +291,13 @@ class ExperimentComponent extends React.Component {
                     }
                     return undefined;
                 },
+                fragmentation_methods: (library) => {
+                    const frag_methods = library.fragmentation_methods;
+                    if (frag_methods && frag_methods.length) {
+                        return frag_methods.sort().join(', ');
```

Comment:
A couple of AirBnB/ESLint corrections needed. All JS identifiers need [camel case](https://github.com/airbnb/javascript#naming--camelCase), and integer truthiness has to be [compared against zero](https://github.com/airbnb/javascript#comparison-operators--equality) in AirBnb. The ESLint plugin will detect this in your editor.

Reply:
Thanks @forresttanaka, I have fixed these two issues